docs/reference/components/sinks/elasticsearch.cue

```diff
@@ -203,6 +203,17 @@ components: sinks: elasticsearch: {
 				syntax: "template"
 			}
 		}
+		mode: {
+			common:      true
+			description: "The type of index mechanism. If `data_stream` mode is enabled, the `bulk_action` is set to `create`."
+			required:    false
+			warnings: []
+			type: string: {
+				default: "regular"
+				examples: ["regular", "data_stream"]
```

Comment:
Maybe we could call `regular` `bulk` instead to make it clearer that the bulk API is being used?

Reply:
That's only tricky because we're still bulk writing either way

Reply:
if we use `regular_bulk`, we should also use `data_stream_bulk` 🤔
if we plan to support the "non bulk" mode, we should add an option `bulk: boolean` I guess 🤔